### ISSUE TITLE
No match found when pattern contains parentheses

### ISSUE THREAD

**Issue by zrh122** [sindresorhus/globby#134]
version `globby@10.0.1`

directory structure:
```
├── tmp(asd)
    └── a.tmp
```
from above directory, run script:
```js
const globby = require('globby');
const paths = globby.sync('./tmp(asd)/a.tmp',  {noext: true});
console.log(paths);
```
the result is empty.

related info: [comment](https://github.com/sindresorhus/globby/issues/74#issuecomment-365784160)

**Comment by sindresorhus** [comment]
This sounds like a `fast-glob` regression. As it was fixed last year: https://github.com/sindresorhus/globby/issues/74#issuecomment-368323132

// @mrmlnc

**Comment by zrh122** [comment]
Thanks for relpy, and I make a mistake. As globby upgrade fast-glob package to v3, I should use new option:
```js
const globby = require('globby');
const paths = globby.sync('./tmp(asd)/a.tmp',  {extglob: false});
console.log(paths);
```
but the result is also empty.

**Comment by mrmlnc** [comment]
Yeah, this is a known issue.

0. https://github.com/micromatch/picomatch/issues/7
1. https://github.com/mrmlnc/fast-glob#why-are-parentheses-match-wrong

**Comment by zrh122** [comment]
So `{extglob: false}` is not work well, what i should do is that:
```js
globby.sync('./tmp\\(asd\\)/a.tmp');
```
🙁but the result is also empty. @mrmlnc 
Maybe you can tell me how to do make things work well. 🙏

**Comment by mrmlnc** [comment]
Unfortunately, it looks like a bug:

```js
fg.sync('styles/\\(tmp\\)/*') → ['styles/(tmp)/a.tmp']
fg.sync('styles/\\(tmp\\)/a.tmp') → []
```

Please, create issue on the `fast-glob` repository.

**Comment by zrh122** [comment]
Thanks for your relpy @mrmlnc , I created an [issue](https://github.com/mrmlnc/fast-glob/issues/223) on the `fast-glob` for it.

**Comment by mrmlnc** [comment]
Should be fixed with `fast-glob@3.1.0`.
